lib/screens/app/ecosystem/overview.dart

```diff
@@ -196,6 +209,16 @@ class _OverviewState extends State<Overview> {
                     onGet,
                   ),
                 ),
+                Consumer<BalanceNotifier>(
```

Comment:
Needs to be in sync with the donation backend - whether we use firebase config for this or we use contracts for that

Reply:
Meaning this button is optional, or there could be a list of them